server/src/main/java/ru/hse/database/entities/Chapter.java

```diff
@@ -0,0 +1,67 @@
+package ru.hse.database.entities;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Objects;
+
+@Entity
+@Table (name = "Chapters")
+public class Chapter {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    @Access(value = AccessType.FIELD)
+    private long chapterId;
+    @Getter
+    @Setter
+    @Column(name = "name")
+    private String chapterName;
+    @Getter
+    @Setter
+    @Column(name = "text")
+    private String textOfChapter;
+
+    @Getter
+    @Setter
+    @Column(name = "test_data")
+    private String testData;
+
+    @Setter
+    @Getter
+    @Column(name = "meta_info")
+    private String metaInfo;
+
+    @Setter
+    @Getter
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "group_host")
+    private Group groupHost;
```

Comment:
mb set Getter and Setter before class declaration and set Setter(AccessLevel.NONE),  Getter(AccessLevel.NONE) to id

Reply:
and same in other classes